acceptanceTests/cypress/integration/page_objects/Traffic.js

```diff
@@ -0,0 +1,9 @@
+export function check(shouldInclude, content, domPathToContainer){
```

Comment:
I think you should rename the file to something that indicates that it's helper methods